ocrd_tool.md

```diff
@@ -10,6 +10,84 @@ services](swagger).
 
 To validate a `ocrd-tool.json` file, use `ocrd ocrd-tool /path/to/ocrd-tool.json validate`.
 
+## Standard parameters
+
+There is a number of parameters common to all processors that MUST be supported by processors.
+
+### `dpi`
+
+Custom DPI to assume for pixel density of images.
+
+MUST default to 300.
+
+### `input-level`
+
+On what level of typography should input images be processed?
+
+Processors MAY define a `default` value.
+
+`enum` MUST be a list of one or more of:
+
+* `page`
+* `block`
+* `line`
+* `word`
+* `glyph`
+
```

Comment:
IMHO we should only have `output-level`. See [arguments here](https://github.com/OCR-D/spec/issues/134#issuecomment-570662568).

Reply:
I removed the section for input-level here and added an example what behavior should be like if the expected input-level isn't provided.

It's probably more productive to continue that discussion on input/output-level in https://github.com/OCR-D/spec/issues/134. I've assgned it to the reviewers here, so let's settle that first and then I'll update the PR.